lib/perl/Genome/Model/PhenotypeCorrelation/Command/CaseControl/Unrelated.pm

```diff
@@ -297,7 +297,8 @@ sub _create_workflow {
                 work_dir => $vep_work_directory,
                 ensembl_annotation_build => $ensembl_annotation_build,
                 log_dir => $log_dir,
-
+                analysis_build => $self->build,
+                analysis_project => $self->build->model->analysis_projects,
```

Comment:
Sure!  Though maybe after this one is settled to avoid making it more complicated. :smile: 

(As an aside, if you press `y` after highlighting a line, Github will make an absolute reference to the line number at its last change revision so that the line-number link won't drift over time.)